yt_dlp/cookies.py

```diff
@@ -347,6 +347,11 @@ def _process_chrome_cookie(decryptor, host_key, name, value, encrypted_value, pa
         if value is None:
             return is_encrypted, None
 
+    # In chrome, session cookies have has_expires and expires_utc set to 0
+    # In Python, cookies that do not expire have expires set to None
+    if not has_expires:
+        expires_utc = None
```

Comment:
Can't we just do this?



```suggestion
    # In chrome, session cookies have expires_utc set to 0
    # In Python, cookies that do not expire have expires set to None
    if not expires_utc:
        expires_utc = None
```


Not asking to change; just trying to understand why we need `has_expires`

Reply:
From what I could see in my chrome database, either should work, you are right

Reply:
I will apply that version locally and see if still all the browsers work

Reply:
<!DOCTYPE html>
Browser | Result
-- | --
Chrome | 
Chromium | (still not working on my mac)
Brave | 
Opera | 
Edge | ✅ 
Vivaldi |

Reply:
I would say the less lines of code the better, if the effect is the same lets just check for 0 😄